### ISSUE TITLE
Home and Library tabs are inaccessible with keyboard navigation (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9194]
## Observed behavior
With the exception of the Bookmark tab, the Library and Home (access to classes) is inaccessible by keyboard navigation.

## Errors and logs
None

## Expected behavior
Users should be able to use their keyboard to navigate throughout the app

## User-facing consequences
Inaccessible to keyboard users

## Steps to reproduce
1. Log in as learner
2. Click on the "Home" tab and try to use the keyboard to navigate through the classes, recent lessons, quizzes, etc.
3. Click on the "Library" tab and try to use the keyboard to navigate

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
After some investigation, I think this is due to default MacBook settings, not a Kolibri issue. After updating my settings I am able to tab through full navigation as expect. 

![firefox-main-nav.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/1718b0df-70f6-4c11-adbf-0adabbada7e3)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
Confirmed - with the correct settings enabled the tab navigation works correctly on Safari, Firefox and Chrome on Mac.
